### ISSUE TITLE
undeploy() reports success even if containers were never up

### ISSUE THREAD

**Issue by gapan** [UCY-LINC-LAB/fogify#5]
In a scenario such as the one described in #3 not all specified containers are started. However, if I run `undeploy()` it reports that all containers have been undeployed (for example 20/20, while only 16/20 were actually deployed in the first place). This is misleading of what actually happens.

**Comment by MosesSymeonidis** [comment]
You are right on that, the reason for both issues is the same. I am working on solving it for both, hopefully, I will have a more stable version at the next release.
